store/postgres/src/metadata.rs

```diff
@@ -883,3 +886,21 @@ fn check_health(conn: &PgConnection, id: &SubgraphDeploymentId) -> Result<(), an
     .map(|_| ())
     .map_err(|e| e.into())
 }
+
+/// Reverts the errors and updates the subgraph health if necessary.
+pub(crate) fn revert_subgraph_errors(
+    conn: &PgConnection,
+    id: &SubgraphDeploymentId,
+    reverted_block: BlockNumber,
+) -> Result<(), anyhow::Error> {
+    use subgraph_error as e;
+
+    delete(
+        e::table
+            .filter(e::subgraph_id.eq(id.as_str()))
+            .filter(sql("lower(block_range) >= $1").bind::<Integer, _>(reverted_block)),
```

Comment:
this should use `block_range::BLOCK_RANGE_COLUMN` as the name of the block range column

Reply:
Done.